web/src/test/java/se/inera/intyg/webcert/web/service/facade/impl/GetPatientFacadeServiceImplTest.java

```diff
@@ -0,0 +1,193 @@
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package se.inera.intyg.webcert.web.service.facade.impl;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import se.inera.intyg.infra.integration.pu.model.Person;
+import se.inera.intyg.infra.integration.pu.model.PersonSvar;
+import se.inera.intyg.infra.integration.pu.services.PUService;
+import se.inera.intyg.schemas.contract.Personnummer;
+import se.inera.intyg.webcert.web.service.facade.patient.GetPatientFacadeServiceImpl;
+import se.inera.intyg.webcert.web.service.intyg.IntygService;
+import se.inera.intyg.webcert.web.service.intyg.dto.IntygServiceResult;
+import se.inera.intyg.webcert.web.service.monitoring.MonitoringLogService;
+import se.inera.intyg.webcert.web.service.receiver.CertificateReceiverService;
+import se.inera.intyg.webcert.web.web.controller.api.dto.IntygReceiver;
+import se.inera.intyg.webcert.web.web.controller.api.dto.IntygTypeInfo;
+import se.inera.intyg.webcert.web.web.controller.facade.dto.PatientResponseDTO;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+public class GetPatientFacadeServiceImplTest {
+
+    @Mock
+    private PUService puService;
+    @Mock
+    private MonitoringLogService monitoringService;
+
+    @InjectMocks
+    private GetPatientFacadeServiceImpl getPatientFacadeService;
+
+    private final static String PATIENT_ID = "191212121212";
+    private final static String FIRSTNAME = "firstname";
+    private final static String LASTNAME = "lastname";
+    private final static String MIDDLENAME = "middlename";
+
+
+    void setupPatient(boolean protectedPerson, boolean testIndicated, boolean deceased, boolean includeMiddleName) {
+        PersonSvar personSvar = createPersonSvar(protectedPerson, testIndicated, deceased, includeMiddleName);
+        doReturn(personSvar).when(puService).getPerson(any());
+    }
+
+    void setupPatient(boolean protectedPerson, boolean testIndicated, boolean deceased) {
+        setupPatient(protectedPerson, testIndicated, deceased, true);
+
+    }
+
+    void setupPatient() {
+        setupPatient(false, false, false, true);
```

Comment:
Not important enough to fix for this code, and perhaps IntelliJ gives a slightly better developer experience, but method signatures like this generally is a bad idea.